wls-admin-service/src/test/java/de/muenchen/oss/wahllokalsystem/adminservice/client/infomanagement/KonfigurierterWahltagClientImplTest.java

```diff
@@ -81,6 +86,67 @@ void should_throwTechnischeWlsException_when_nonWlsExceptionIsThrownFromAKonfigu
 
             Assertions.assertThatException().isThrownBy(() -> unitUnderTest.postKonfigurierterWahltag(konfigurierterWahltagModel)).isSameAs(mockedWlsException);
         }
+
+        @Test
+        void should_notResetWahlen_when_wahlIsNotActive() {
+            val nowDate = LocalDate.now();
+            val konfigurierterWahltagModel = new KonfigurierterWahltagModel(nowDate, "1-2-3", true, "123");
+            val mockedKonfigurierterWahltagDTO = new KonfigurierterWahltagDTO().wahltag(nowDate).wahltagID("1-2-3").wahltagStatus(
+                    KonfigurierterWahltagDTO.WahltagStatusEnum.INAKTIV).nummer("123");
+
+            Mockito.when(konfigurierterWahltagClientMapper.toDto(konfigurierterWahltagModel)).thenReturn(mockedKonfigurierterWahltagDTO);
+
+            unitUnderTest.postKonfigurierterWahltag(konfigurierterWahltagModel);
+
+            Mockito.verify(konfigurierterWahltagControllerApi).setKonfigurierterWahltag(mockedKonfigurierterWahltagDTO);
+            Mockito.verifyNoInteractions(wahlenControllerApi);
+        }
+    }
+
+    @Nested
+    class GetKonfigurierteWahltage {
+
+        @Test
+        void should_callApi_when_konfigurierteWahltageIsCalled() throws Exception {
+            val nowDate = LocalDate.now();
+            val konfigurierterWahltagModel = new KonfigurierterWahltagModel(nowDate, "1-2-3", true, "123");
+            val mockedKonfigurierterWahltagDTO = new KonfigurierterWahltagDTO().wahltag(nowDate).wahltagID("1-2-3").wahltagStatus(
+                    KonfigurierterWahltagDTO.WahltagStatusEnum.AKTIV).nummer("123");
+
+            Mockito.when(konfigurierterWahltagControllerApi.getKonfigurierteWahltage()).thenReturn(List.of(mockedKonfigurierterWahltagDTO));
+            Mockito.when(konfigurierterWahltagClientMapper.toModel(mockedKonfigurierterWahltagDTO)).thenReturn(konfigurierterWahltagModel);
+
+            unitUnderTest.getKonfigurierteWahltage();
+
+            Mockito.verify(konfigurierterWahltagControllerApi).getKonfigurierteWahltage();
+        }
+
+        @Test
+        void should_throwTechnischeWlsException_when_exceptionIsThrownByApi() throws Exception {
+            val mockedWlsException = TechnischeWlsException.withCode("000").buildWithMessage("Communication with KonfigurierteWahltageApi failed");
+
+            Mockito.doThrow(mockedWlsException).when(konfigurierterWahltagControllerApi).getKonfigurierteWahltage();
+
+            Assertions.assertThatException().isThrownBy(() -> unitUnderTest.getKonfigurierteWahltage()).isSameAs(mockedWlsException);
+        }
+
+        @Test
+        void should_returnNull_when_apiReturnsNull() throws Exception {
```

Comment:
Die Beschreibung passt nicht zum Testinhalt.

```suggestion
        void should_returnEmptyList_when_apiReturnsNull() throws Exception {
```